incrementals-publisher/README.adoc

```diff
@@ -0,0 +1,79 @@
+= Incremental Build Publisher
+
+
+This Azure Function is responsible for bouncing incrementally built (see
+link:https://github.com/jenkinsci/jep/tree/master/jep/305[JEP-305]) core and
+plugin artifacts into the incrementals Maven repository (see
+link:https://github.com/jenkins-infra/iep/tree/master/iep-009[IEP-9]).
+
+
+[[env]]
+== Environment Variables
+
+|===
+| Variable | Description
+
+| `GITHUB_TOKEN`
+| A GitHub OAuth/Personal Access Token which can be used for GitHub API queries
```

Comment:
preferably also `repo:status` (best to be explicit about what ought to be granted)